File: backend/profile-service/src/main/java/com/example/profileservice/profile/dto/CreateProfileDto.java
package com.example.profileservice.profile.dto;

import com.example.profileservice.profile.entity.ProfileType;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class CreateProfileDto {

    private Long memberId;
    private ProfileType profileType;
    private String value;
    private String createdDate;
    private String updatedDate;
}

File: backend/profile-service/src/main/kotlin/com/example/profileservice/profile/dto/CreateProfileDto.kt
package com.example.profileservice.profile.dto

import com.example.profileservice.profile.entity.ProfileType

data class CreateProfileDto(
    var memberId: Long? = null,
    var profileType: ProfileType? = null,
    var value: String? = null,
    var createdDate: String? = null,
    var updatedDate: String? = null,
)
